back/engines/commercial/flag_inappropriate_content/app/services/flag_inappropriate_content/extensions/nlp/api.rb

```diff
@@ -0,0 +1,23 @@
+module FlagInappropriateContent
```

Comment:
Nitpicking, but wouldn't it be better to create a service that inherits from `NLP::Api` instead?

Reply:
I opted to extend the existing service, so we have one NLP service that can handle all the endpoints. I don't have a strong opinion about it, but being able to use all endpoints with the same instantiation of the service seems like a small advantage to me. Are there advantages of having several services, each inheriting the core service instead?